migrations/5_init_protocol.js

```diff
@@ -35,42 +31,25 @@ module.exports = function(deployer, network) {
         ])
     }).then(() => {
         // Register Verifier
-        if (network === "development") {
+        if (network === "development" || network === "parityDev" || network === "gethDev") {
             return controller.setContract(ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["Verifier"])), IdentityVerifier.address)
         } else if (network === "lpTestNet") {
             return controller.setContract(ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["Verifier"])), LivepeerVerifier.address)
         } else {
             return controller.setContract(ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["Verifier"])), OraclizeVerifier.address)
         }
     }).then(() => {
-        // Register upgradeable proxy target contracts
+        // Register contracts
         return Promise.all([
-            controller.setContract(ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["BondingManagerTarget"])), BondingManager.address),
-            controller.setContract(ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["JobsManagerTarget"])), JobsManager.address),
-            controller.setContract(ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["RoundsManagerTarget"])), RoundsManager.address)
+            controller.setContract(ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["BondingManager"])), BondingManager.address),
+            controller.setContract(ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["JobsManager"])), JobsManager.address),
+            controller.setContract(ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["RoundsManager"])), RoundsManager.address)
         ])
     }).then(() => {
-        // Deploy proxy contracts
         return Promise.all([
-            ManagerProxy.new(Controller.address, ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["BondingManagerTarget"]))),
-            ManagerProxy.new(Controller.address, ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["JobsManagerTarget"]))),
-            ManagerProxy.new(Controller.address, ethUtil.bufferToHex(ethAbi.soliditySHA3(["string"], ["RoundsManagerTarget"])))
-        ])
-    }).then(proxies => {
```

Comment:
Is the idea that after you debug the proxy implementation you'd just change this migration basically back to the original state?

Reply:
Pretty much, just taking it out for now to not block any efforts to do end to end testing on the test net